### PR TITLE
Update to 1.3.8 + readme additional informations 

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- Behind the upstream application

## Solution
- Updated to last version 1.3.8
- Update carddav to 3.0.3
- Added informations in the README 
- Fix serveral errors from package_linter: [YEP-2.4] and [YEP-1.3]

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
- [x] **Approval (LGTM)** : JimboJoe
- **CI succeeded** : 

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/roundcube_ynh%20PR50%20(Official_fork)/)
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

File: check_process [hunk 1]
```diff
@@ -12,7 +12,6 @@
 		setup_private=0
 		setup_public=0
 		upgrade=1
-		upgrade=1	from_commit=8d0acb6953a4c4c2206985c7d9881c5b7b637ebd
```
Comment (maniackcrudelis) [inline check_process]: Why do you remove this test ?

Comment (yalh76) [reply check_process]: Because it was an upgrade to an older version... I don't think it can work...

Comment (maniackcrudelis) [reply check_process]: An upgrade __from__ an older version. That's the point of this test.
If your upgrade script doesn't work when upgrading from an older version, then it has to be fixed.

Comment (yalh76) [reply check_process]: OK then, misunderstanding of that functionality, So I haven't tested ^^
I roll back the check_process

**Comment by maniackcrudelis** [comment]
Globally agree with this PR.
Anyway, I'm currently working on this app as well.

**Comment by JimboJoe** [comment]
Will be merged in 3 days (no code change since @maniackcrudelis 's review)

**Pull request merged**